From d93d189ce3155576592d329a2d02976b6360c527 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Mon, 26 Jun 2017 14:17:48 +0800
Subject: [PATCH 2026/5242] MLK-15308 arm64: dts: imx8mq-evk: add post delay
 for BT modem

commit  a456613405083d57411a47cdc501ff9262395754 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8MQ evk board install Murata 1CX WIFI/BT module, 1CX Bluetooth
requires the post delay after BT_REG power on.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Pandy Gao <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 0092b01..61523ff 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -32,7 +32,8 @@
 	modem_reset: modem-reset {
 		compatible = "gpio-reset";
 		reset-gpios = <&gpio3 5 GPIO_ACTIVE_LOW>;
-		reset-delay-us = <1000>;
+		reset-delay-us = <2000>;
+		reset-post-delay-ms = <40>;
 		#reset-cells = <0>;
 	};
 
-- 
1.7.9.5

